dashboard/src/shared/schema.ts

```diff
@@ -67,15 +60,24 @@ export function retrieveBasicFormParams(
 }
 
 // orderParams conveniently structure the parameters to satisfy a parent-children relationship even if
-// those parameters doesn't have that relation in the source
+// those parameters doesn't have that relation in the source. This is only used when a parameter
+// enables/disables another.
+// CAVEAT: It only works with one level of depth
```

Comment:
Looks fixable in the future if/when we need to (depending on what we find in charts). Thanks for the note.